src/utils/domains.ts

```diff
@@ -13,7 +13,7 @@ export const findDomainByNativeId = (
   }
   return domains
     .filter(notNull)
-    .find(({ nativeId }) => nativeId === domainNativeId);
+    .find(({ nativeId }) => nativeId === Number(domainNativeId));
```

Comment:
isn't domainNativeId already a number or undefined/null? if we cast `null` or `undefined` to Number we'll get 0, so this may give us a headache in the future

Reply:
It was transformed into string in `useTeams` method when user clicked "Edit team" button.